frontend/src/components/common/select/select.tsx

```diff
@@ -13,18 +14,26 @@ type Props = {
   control: FormControl;
   errors: FormControlErrors;
   name: string;
-  options: SelectorOptions[];
+  options: SelectorOptions<string | number>[];
```

Comment:
```suggestion
  options: SelectorOptions<string | number>[];
```
rename `SelectorOptions` to `SelectorOption` (don't forget about file name)